roles/openshift_cfme/tasks/nfs.yml

```diff
@@ -1,6 +1,13 @@
 ---
 # Tasks to statically provision NFS volumes
 # Include if not using dynamic volume provisioning
+
+- name: Set openshift_cfme_nfs_server fact
+  when: openshift_cfme_nfs_server is not defined
+  set_fact:
+    # Hostname/IP of the NFS server. Currently defaults to first master
+    openshift_cfme_nfs_server: "{{ groups.nfs.0 }}"
```

Comment:
I think the correct value for `openshift_cfme_nfs_server` should be `oo_nfs_to_config.0` That might also help cut down on failures due to un-defined groups. That is because `playbooks/byo/openshift-cfme/config.yml` calls the `initialize_groups.yml` and `evaluate_groups.yml` task files which define our canonical group names (and members) so that playbooks/roles can run independent of how things are named in inventories.